### ISSUE TITLE
CustomSearch API typed incorrectly

### ISSUE THREAD

**Issue by johnnybarton411** [Maxim-Mazurok/google-api-typings-generator#683]
When running `gapi.client.load('customsearch', 'v1')`, the `gapi.client` Window object is updated to include `gapi.client.search`, versus the expected type `gapi.client.customsearch` expected by the type definitions that this generator produces.

**Comment by Maxim-Mazurok** [comment]
Hello John!
Thank you for raising the issue.
This is a bug in the older generator that was addressed in #655 
PR to update DefinitelyTyped was already opened a month ago, but it seems like the bot doesn't pick it up: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62106 I've pinged DT maintainers and going to try another workaround.
As part of this PR we're migrating from `@types/gapi.client.customsearch` to `@types/gapi.client.customsearch-v1` naming convention to support different versions. Old types won't get updates and will be removed eventually.
Also I've double-checked that `gapi.client.load('customsearch', 'v1')` creates `gapi.client.search` by adding this test: https://github.com/Maxim-Mazurok/gapi/commit/f139b655854e43f8c385d2b991d4678d184ceac9

**TL;DR**: use [`@maxim_mazurok/gapi.client.customsearch-v1`](https://www.npmjs.com/package/@maxim_mazurok/gapi.client.customsearch-v1) package from npm. Once https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62106 is merged - replace it with `@types/gapi.client.customsearch-v1`.

Let me know if you'll have any other questions, cheers!

**Comment by johnnybarton411** [comment]
Thank you for your response (and for creating/maintaining this awesome repo)!  Workaround makes sense, thank you.
